BackwardConfig.cmake

```diff
@@ -122,7 +122,7 @@ foreach(def ${BACKWARD_DEFINITIONS})
 	message(STATUS "${def}")
 endforeach()
 
-find_path(BACKWARD_INCLUDE_DIR backward.hpp PATHS ${CMAKE_CURRENT_LIST_DIR})
+set(BACKWARD_INCLUDE_DIR "${CMAKE_CURRENT_LIST_DIR}")
```

Comment:
do you know where CMAKE_CURRENT_LIST_DIR is coming from?

I am afraid changing this might break the build for some people. At this point, there is more cmake code than there is C++ code in this library. This is beyond ridiculous.

Reply:
[`CMAKE_CURRENT_LIST_DIR`] is a variable handled by CMake. It was [introduced by CMake 2.8.3].

[`CMAKE_CURRENT_LIST_DIR`]: https://cmake.org/cmake/help/v3.0/variable/CMAKE_CURRENT_LIST_DIR.html
[introduced by CMake 2.8.3]: https://cmake.org/pipermail/cmake/2010-November/040578.html

Reply:
As `backward-cpp` CMakeLists.txt starts with `cmake_minimum_required(VERSION 2.8.12)`, it is safe to assume that the variable exists.